src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -948,12 +1218,12 @@ end
 			warn(err)
 		end)
 	```
-	
+
 	@within AnalyticsService
 	@param remote RemoteConfig -- The name, default value, and context of the remote configuration
-	@return string
+	@return Promise
 ]=]
-function AnalyticsService:GetRemoteConfig(remote: RemoteConfig): { [any]: any }
+function AnalyticsService:GetRemoteConfig(remote: RemoteConfig)
```

Comment:
```suggestion
function AnalyticsService:GetRemoteConfig(remote: RemoteConfig): {[any]: any}
```
Outside of a custom defined promise type, {[any]: any} is the closest to a return type for a promise in my opinion.